test/connect-backbone-to-react.test.js

```diff
@@ -200,6 +200,24 @@ describe('connectBackboneToReact', function() {
     });
   });
 
+  describe('when mounted with an undefined model', function() {
+    afterEach(function() {
+      wrapper.unmount();
+    });
+
+    function mapEmptyModel({ user }) {
+      return { user };
+    }
+
+    it('should mount and unmount the component successfully', function() {
+      const ConnectedTest = connectBackboneToReact(mapEmptyModel)(TestComponent);
```

Comment:
What happens if we don't pass in a map function and just let the library use the default mapping? i.e. if you do `connectBackboneToReact()(TestComponent);`?

Reply:
If we do that it looks like the `defaultMapModelsToProps` function actually fails. We could probably add more safety checks when we make calls on the models like `model.toJSON()`.

Though, in order to repro the specific bug we need to pass our own mapping function.